src/components/dashboard/customAdminPage.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import { experimentalStyled as styled } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Paper from "@mui/material/Paper";
+import Grid from "@mui/material/Grid";
+import DashboardChart from "./CustomPieChart";
+import Typography from "@mui/material/Typography";
+import "../style/styling.css";
+
+//using a custom object Item.
+/**
+ * Item is used to display typography 
+ */ 
+const Item = styled(Paper)(({ theme }) => ({
+    backgroundColor: theme.palette.mode === "dark" ? "#1A2027" : "#fff",
```

Comment:
These colors should be moved into the theme and accessed from the `theme` prop instead of hardcoding them

Reply:
I removed lines 15 and 16 as they were not necessary, given that there are no plans for implementing a dark or light mode.